libraries/abstractions/ble_hal/include/bt_hal_manager.h

```diff
@@ -59,16 +59,10 @@
  */
 #define btPATCH_VERSION    1
 
-/**
- * @brief  Help functions to convert version to string.
- */
-#define STR_HELPER( x, y, z )    # x "." # y "." # z
-#define STR( x, y, z )           STR_HELPER( x, y, z )
-
 /**
  * @brief  Stringified version number.
  */
-#define btSTRING_VERSION                        STR( btMAJOR_VERSION, btMINOR_VERSION, btPATCH_VERSION )
+#define btSTRING_VERSION                        # btMAJOR_VERSION "." # btMINOR_VERSION "." # btPATCH_VERSION
```

Comment:
Can you confirm this works ? The stringification of macro requires it to be passed as an argument and requires one level of indirection as mentioned here:
https://gcc.gnu.org/onlinedocs/gcc-4.8.5/cpp/Stringification.html

Reply:
Added back macros and appended "bt" in front of macro to address ACS team concerns